README.md

```diff
@@ -649,7 +649,7 @@ maintaining the Node.js project.
 * [marsonya](https://github.com/marsonya) -
   **Akhil Marsonya** <<akhil.marsonya27@gmail.com>> (he/him)
 * [Mesteery](https://github.com/Mesteery) -
-  **Mestery** <<mestery@pm.me>>
+  **Mestery** <<mestery@protonmail.com>>
```

Comment:
Do you want to add pronouns so it's synced with the line above? https://github.com/nodejs/node/blob/eda54bae16007a391283a8ee0d52bcd6ee311b5c/README.md#L384